deployment/v3/mosip/credential-feeder/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Installs idrepo
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=credentialfeeder
+CHART_VERSION=12.0.2
```

Comment:
In the helm chart, the chart version is set to 1.0.0. 

please get it confirmed by @ckm007